src/samd_platform.cpp

```diff
@@ -1,6 +1,10 @@
+#if(defined(__SAMD21E17A__) || \
+    defined(__SAMD21G18A__) || \
+    defined(__SAMD21E18A__) || \
+    defined(__SAMD21J18A__))
+
 #include "samd_platform.h"
 
-#ifdef ARDUINO_ARCH_SAMD

```

Comment:
Any reason to not also put
 #if (defined(__SAMD21E17A__) || \
     defined(__SAMD21G18A__) || \
     defined(__SAMD21E18A__) || \
     defined(__SAMD21J18A__)) 
here?

Reply:
No, copy paste ended up at the start of the file. I will move them after the include samd_platform.h to be consistent with the other files.